code/html/custom.js

```diff
@@ -820,57 +830,13 @@ function initRelays(data) {
 
 }
 
-function initCheckboxes() {
-
-    var setCheckbox = function(element, value) {
-        var container = $(".toggle-container", $(element));
-        if (value) {
-            container.css("-webkit-clip-path", "inset(0 0 0 50%)");
-            container.css("clip-path", "inset(0 0 0 50%)");
-            container.css("backgroundColor", "#00c000");
-        } else {
-            container.css("-webkit-clip-path", "inset(0 50% 0 0)");
-            container.css("clip-path", "inset(0 50% 0 0)");
-            container.css("backgroundColor", "#c00000");
-        }
-    }
-
-    $(".checkbox-container")
-
-        .each(function() {
-            var status = $(this).next().prop('checked');
-            setCheckbox(this, status);
-        })
-        .off('click')
-        .on('click', function() {
-
-            var checkbox = $(this).next();
-
-            var status = checkbox.prop('checked');
-            status = !status;
-            checkbox.prop('checked', status);
-            setCheckbox(this, status);
-
-            if ("relay" == checkbox.attr('name')) {
-                var id = parseInt(checkbox.attr('data'), 10);
-                doToggle(id, status);
-            }
-
-        });
-
-}
-
 function createCheckboxes() {
 
     $("input[type='checkbox']").each(function() {
 
-        var text_on = $(this).attr("on") || "YES";
-        var text_off = $(this).attr("off") || "NO";
```

Comment:
Sorry dude, I'm not a huge fan of hacks and workarounds so no div + proxies ). I generally like keeping things as simple as they can be.

Reply:
Well... does it not work only because label does the proxy by itself internally?:)
Relays are ok now. Plus, I should've thought of rules order issue...